bashlex/tokenizer.py

```diff
@@ -47,7 +47,6 @@ class tokentype(enum.Enum):
     COPROC = 15
     COND_START = 16
     COND_END = 17
-    COND_ERROR = 18
```

Comment:
Can you comment it out instead and leave a comment with a link to the issue on github? Thanks.